tools/stats/import_test_stats.py

```diff
@@ -8,19 +8,20 @@
 from typing import Any, Callable, Dict, List, Optional, cast
 from urllib.request import urlopen
 
-# PYTORCH_IGNORE_DISABLED_ISSUES should only be set during CI (along with IN_CI) as a
-# comma-separated list of issue numbers. The intent is to re-enable any disabled tests
-# associated with the issues in this list.
-#
-# There is normally no reason to use this locally as the disabled tests list should not
-# affect your local development and every test should be enabled. If for whatever reason
-# you would like to use this during local development, please note the following caveat:
-#
-# Whenever you set OR reset PYTORCH_IGNORE_DISABLED_ISSUES, you should delete the existing
-# .pytorch-disabled-tests.json and redownload/parse the file for your change to apply, as
-# PYTORCH_IGNORE_DISABLED_ISSUES is used during the parsing stage. To download the files,
-# run test/run_test.py with IN_CI=1.
-IGNORE_DISABLED_ISSUES: List[str] = os.getenv('PYTORCH_IGNORE_DISABLED_ISSUES', '').split(',')
+def get_disabled_issues() -> List[str]:
```

Comment:
@janeyx99: is this logic tested anywhere? I noticed that it doesn't work for the common case where you put a whole issue link (as opposed to just the issue number) in the PR body. Personally I do that and never use the issue number, which means this logic would not work for me.

Reply:
There is tools/test/test_stats.py, but `get_disabled_issues` is not tested there. Mind adding it to the list ;)

Reply:
That's a good point, I only tested locally but didn't add any unit tests, and we should for at least the regex part.